src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml

```diff
@@ -75,20 +75,15 @@
                           IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"
                            />
 
-            <TextBlock Name="PowerRename_Toggle_MaxDispListNum"
-                x:Uid="PowerRename_Toggle_MaxDispListNum"
-                Margin="{StaticResource SmallTopMargin}"
-                Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
-
-            <muxc:NumberBox SpinButtonPlacementMode="Compact"
+            <muxc:NumberBox x:Uid="PowerRename_Toggle_MaxDispListNum"
+                            SpinButtonPlacementMode="Compact"
                             HorizontalAlignment="Left"
-                            Margin="{StaticResource HeaderTextTopMargin}"
+                            Margin="{StaticResource SmallTopMargin}"
                             Value="{x:Bind Mode=TwoWay, Path=ViewModel.MaxDispListNum}"
                             Minimum="0"
                             Width="240"
                             Maximum="20"
-                            IsEnabled="{ x:Bind Mode=OneWay, Path=ViewModel.GlobalAndMruEnabled}"
-                            AutomationProperties.LabeledBy="{Binding ElementName=PowerRename_Toggle_MaxDispListNum}"/>
```

Comment:
Any reason to remove this `AutomationProperties.LabeledBy` property ?

Reply:
It's no longer needed, right?

Before, we had to label the seperate TextBlock as the value to be read by the Narrator for the NumberBox. With this PR we removed that TextBlock because we can use the built in Header property.

Reply:
@niels9001 I verified it works as expected.